gcp/helmfile.yaml

```diff
@@ -15,8 +15,7 @@
 environments:
   default:
     values:
-      - workload_identity_project_id: "%GCLOUD_PROJECT%"
```

Comment:
were there any scripts or steps in the readme to replace these `%%` strings?

Reply:
Not anymore.